src/rgw/rgw_civetweb.cc

```diff
@@ -129,7 +129,14 @@ int RGWCivetWeb::init_env(CephContext *cct)
     }
     *dest = '\0';
 
-    env.set(buf, value);
+    string sanitized_value = value;
```

Comment:
this works, but should we be considering the entire header?

Reply:
yes, pushed a change that moves the sanitization to top of the header elements processing loop to cover all the header elements.